src/breeze/actions/emails/index.cr

```diff
@@ -0,0 +1,6 @@
+class Breeze::Emails::Index < BreezeAction
+  get "/emails" do
+    emails = Carbon::Email.email_classes.map(&.name)
```

Comment:
I think it might be better to only show emails that have previews set up. Here's another approach that *might* work

```
class CarbonPreviews
  def previews
    {
      welcome_email: WelcomeEmail.new(user: UserQuery.first),
      some_other_email: "whatever"
    } 
  end
end
```

The key for the NamedTuple can be the "id". The Emails::Index can iterate through them. When clicked it goes to Emails::Show which outputs the email from the given key. I think this would be pretty simple to code, requires less meta-programming, and is super simple to understand and extend (IMO)

Thoughts?

Reply:
I like that. Sorta of goes to what @matthewmcgarvey was saying too. It feels cleaner than what I was doing.